src/js/site/insightsTableScroll.test.js

```diff
@@ -0,0 +1,75 @@
+import { insightsTableScroll } from './insightsTableScroll';
+
+window.$ = require('src-api/source/javascripts/lib/_jquery.js');
+
+document.body.innerHTML = '<div id="overallSuccesRate-definition">' + '</div>';
```

Comment:
Is there a reason you are using string concatenation here? It seems that there is no content in this html that you are concat'ing between the two divs? IN which case you could do this?

`'<div id="overallSuccesRate-definition"></div>';`  
?

Reply:
At first I had some items in the middle. Simply just didn't clean up thanks :)